packages/forma-36-website/src/components/Navigation.js

```diff
@@ -45,16 +45,17 @@ const styles = {
       ${tokens.transitionEasingDefault};
 
     &:hover {
-      background-color: ${tokens.colorElementLight};
+      background-color: ${tokens.colorElementLightest};
     }
   `,
 
   linkActive: css`
-    background-color: ${tokens.colorBlueLight};
-    color: ${tokens.colorWhite};
+    background-color: ${tokens.colorElementLight};
```

Comment:
I am not sure how to handle the navigation when it comes to colors, we could get some support from designers here I think. For now, I wanted the navigation state to pass the accessibility requirements. (Currently, the active item does not pass it).
I've decided to have an active light gray background and text to be colorTextDark and bold. WDYT?

Reply:
Can you add screenshots for the lazy (like me :D)

Reply:
I think we should just use the regular active blue on hover - so basically not giving active items a hover state

Reply:
So this is what I had in mind. (I don't like the blue active state, apart of the fact that now it's not passing the accessibility requirements):

<img width="310" alt="Screenshot 2020-12-22 at 13 17 57" src="https://user-images.githubusercontent.com/4272331/102887980-2dbb6a00-4458-11eb-9ec0-4e0cc86590f0.png">
<img width="312" alt="Screenshot 2020-12-22 at 13 18 08" src="https://user-images.githubusercontent.com/4272331/102887982-2f852d80-4458-11eb-8ab5-8c7a63bdb092.png">

Reply:
I think blue is more consistent - from what I remember it was grey a while back and there was a deliberate decision to make it blue, so changing it back to grey seems odd (but I totally get the accessibility part)

So I think just using `#2e75d4` / `colorPrimary` as the background makes the most sense to me (as in removing that hover state and just using `colorPrimary`

Reply:
hmm, ok, I see. Ok, I will make it `colorPrimary` then. Although I think it is not the best choice for navigation. (in general, I think we just have a too strong blue appearance on the website.)

Reply:
@m10l done, I rolled back to blue, hover the same as it was, blue on active and no active hover. ✅